spec/features/user_index_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe 'User index', type: :feature do
+  before :each do
+    @user1 = User.create(name: 'John Doe', photo: 'https://example.com/john-doe.jpg', bio: 'Software Engineer',
+                         post_counter: 5)
+    @user2 = User.create(name: 'Jane Smith', photo: 'https://example.com/jane-smith.jpg', bio: 'UX Designer',
+                         post_counter: 3)
+    visit root_path
+  end
+
```

Comment:
- [ ] Hi, after checking your test, I noticed that two featured tests for `User` **index page** are missing:
- Tests that asserts you can see the username of all other users.
- Tests that asserts you can see the profile picture for each user.

Can you add them? Because it is part of the requirements